assets/theme-supplement.css

```diff
@@ -529,4 +539,25 @@
   }
 }
 
+.sc-review {
+  display: grid;
+}
+
+.sc-review_wrapper {
+  gap: 20px;
+  padding: 50px;
+  height: -webkit-fill-available;
+  text-align: center;
+  border: 1px solid rgba(128, 128, 128, 0.16);
+  box-shadow: 0 1px 4px #8080801c;
```

Comment:
You can also remove this property and replace it with `sc-shadow` utility class instead.